core/src/main/scala/org/apache/spark/ui/exec/ExecutorsPage.scala

```diff
@@ -52,68 +53,80 @@ private[ui] class ExecutorsPage(
   private val listener = parent.listener
 
   def render(request: HttpServletRequest): Seq[Node] = {
-    val storageStatusList = listener.storageStatusList
+    val activeStorageStatusList = listener.activeStorageStatusList
+    val activeExecutorsTable = listingExecTable(activeStorageStatusList, true)
+    val deadStorageStatusList = listener.deadStorageStatusList
+    val deadExecutorsTable = listingExecTable(deadStorageStatusList, false)
+    val content =
+      <span>
+        <h4>ActiveExecutors({activeStorageStatusList.size})</h4> {activeExecutorsTable}
+        <h4>DeadExecutors({deadStorageStatusList.size})</h4> {deadExecutorsTable}
+      </span>
```

Comment:
can you put `{ }` around this? It makes the DSL a little less scary because it guarantees that all the nodes are concatenated correctly together